### PR TITLE
New observableSwitch operator

### PR DESCRIPTION
**Pull request by jacek-marchwicki**

### CODE REVIEW

File: rx-extensions/src/test/java/com/appunite/rx/operators/MoreOperatorsObservableSwitch.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+import rx.Observable;
+import rx.Scheduler;
+import rx.functions.Func1;
+import rx.observers.TestSubscriber;
+import rx.schedulers.Schedulers;
+import rx.schedulers.TestScheduler;
+import rx.subjects.PublishSubject;
+
+public class MoreOperatorsObservableSwitch {
```
Comment (m-zagorski) [inline rx-extensions/src/test/java/com/appunite/rx/operators/MoreOperatorsObservableSwitch.java]: MoreOperatorsObservableSwitchTest?

Comment (jacek-marchwicki) [reply rx-extensions/src/test/java/com/appunite/rx/operators/MoreOperatorsObservableSwitch.java]: oops

File: rx-extensions/src/main/java/com/appunite/rx/operators/MoreOperators.java [hunk 2]
```diff
@@ -502,4 +505,62 @@ public void call() {
...
+            this.observables = observables;
+        }
+
+        State() {
+            this(Collections.<T, Observable<R>>emptyMap(), Collections.<Observable<R>>emptyList());
+        }
+    }
+
+    /**
+     * Operator use func to covert T element in to list of R bye applying Observable
```
Comment (m-zagorski) [inline rx-extensions/src/main/java/com/appunite/rx/operators/MoreOperators.java]: by instead of bye

**Comment by m-zagorski** [comment]
lgtm

File: rx-extensions/src/main/java/com/appunite/rx/operators/MoreOperators.java (hunk 2 above)
Comment (jacek-marchwicki) [reply rx-extensions/src/main/java/com/appunite/rx/operators/MoreOperators.java]: done

**Pull request merged**